aten/src/ATen/native/Math.h

```diff
@@ -1247,7 +1240,7 @@ inline std::tuple<const T*, size_t> chebyshev_coefficients_i0e_B() {
 };
 
 template <typename T>
-inline typename std::enable_if<std::is_same<double, T>::value, std::tuple<const T*, size_t>>::type
+static inline typename std::enable_if<std::is_same<double, T>::value, std::tuple<const T*, size_t>>::type
```

Comment:
What does declaring these static do?

Reply:
This header is included in multiple files. With static, we are assuring the compiler that this function will only be used in current translation unit. Without static,(I think) compiler is not free to remove the code even if it is unused as some other translation unit may refer to it.

Reply:
Without static, the assembly still has unused function

https://godbolt.org/z/Td8exxYMP

With static, it is removed

https://godbolt.org/z/Y7v5EMh6h